dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deals-cancellation-repo.test.ts

```diff
@@ -47,6 +47,19 @@ describe('tfm-deals-cancellation-repo', () => {
       await expect(TfmDealCancellationRepo.findDealCancellationByDealId(dealId)).rejects.toThrow(new DealNotFoundError(dealId.toString()));
     });
 
+    it('returns an empty object if a deal is found without a cancellation', async () => {
```

Comment:
Can we add an assertion with an invalid `dealId` please.

Reply:
Looks like Mark has added a test for this in👍